### PR TITLE
Update deploy guide

### PR DESCRIPTION
**Pull request by arukshani**
$subject

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ If the source file contains at least one entrypoint, it can be executed using th
 $ ballerina run foo.bal
 ```
 
-You can compile a source file with an entrypoint into a linked binary that has a `.jar` extension.
+You can compile a source file with an entrypoint into an executable jar.
     
 ```bash
-$ ballerina build a/b/c/foo.bal [-o outputfilename.jar]
+$ ballerina build foo.bal [-o outputfilename.jar]
```
Comment (warunalakshitha) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
$ ballerina build [-o outputfilename.jar] foo.bal 
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 2]
```diff
@@ -76,18 +76,9 @@ public function main() {
 }
 ```
 
-The config key is `hello.user.name`. To pass a value to this config from the CLI, we can run the following command. The `-e` argument passes the key and value to the program.
+The config key is `hello.user.name`. To pass a value to this config from the CLI, we can run the following command. The `--` passes the key and value to the program.
```
Comment (warunalakshitha) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: '--' actually have different meaning where we use to terminate the config args. So after thet even you have --something will take as args.

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 3]
```diff
@@ -107,18 +98,9 @@ If `ballerina.conf` resides in the same directory as `main.bal`, `balllerina run
 $ ballerina run main.bal
 Hello, Ballerina !
 ```
-To explicitly specify a configuration file, use either the `--config` or the `-c` flag. The path to the configuration file can be either an absolute or a relative path. 
+To explicitly specify a configuration file, use either the `--b7a.config.file` flag. The path to the configuration file can be either an absolute or a relative path.
```
Comment (warunalakshitha) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
To explicitly specify a configuration file, use the `--b7a.config.file` property. The path to the configuration file can be either an absolute or a relative path. 
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md (hunk 2 above)
Comment (warunalakshitha) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
The config key is `hello.user.name`. To pass a value to this config from the CLI, we can use `--key=value` format as the following command. 
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ A Ballerina application can either be:
 
 1. A [`public function`](/learn/by-example/functions-as-entry-points.html) including [`main()`](/learn/by-example/hello-world.html) function that runs as a terminating process.
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: We don't have the functions-as-entry-points example now. 

Shall we change L4 to 
"A Ballerina application can have: "

And change this to 
```suggestion
1. A [`main()`](/learn/by-example/the-main-function.html) function that runs as a terminating process.
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ A Ballerina application can either be:
 
 1. A [`public function`](/learn/by-example/functions-as-entry-points.html) including [`main()`](/learn/by-example/hello-world.html) function that runs as a terminating process.
 
-2. A [`service<>`](/learn/by-example/hello-world-service.html), which is a hosted non-terminating process.
+2. A [`service`](/learn/by-example/hello-world-service.html), which is a hosted non-terminating process.
 
 Both of these are considered "entrypoints" for program execution.
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
Both of these are considered as entry points for program execution. 
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md (hunk 1 above)
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: This doesn't work atm. Need to swap the option

```suggestion
$ ballerina build [-o outputfilename.jar] foo.bal
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 6]
```diff
@@ -22,15 +22,15 @@ If the source file contains at least one entrypoint, it can be executed using th
...
     
 ```bash
-$ ballerina build a/b/c/foo.bal [-o outputfilename.jar]
+$ ballerina build foo.bal [-o outputfilename.jar]
 ```  
 
 And you can run `.jar` files directly:
 ```bash
-$ ballerina run filename.jar
+$ ballerina run filename-executable.jar
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
$ ballerina run filename.jar
```

The `-executable` suffix was removed.

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 7]
```diff
@@ -48,8 +48,8 @@ $ ballerina build <module-name>
 
 Options for running programs with entrypoints in a project:  
 ```bash
-$ ballerina run main.jar  
-$ ballerina run target/main.jar
+$ ballerina run main.bal
+$ ballerina run main-executable.jar
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
$ ballerina run main.jar
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 8]
```diff
@@ -230,40 +215,43 @@ Now your code is ready to generate deployment artifacts. In this case it is a Do
 ```bash
 $ ballerina build hello_world_docker.bal  
 Compiling source
-    hello_world_docker.bal
+        hello_world_docker.bal
+Generating executables
+        hello_world_docker-executable.jar
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: Shall we recheck the `-executable` suffix everywhere?

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md (hunk 2 above)
Comment (arukshani) [reply website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: what do you want  me to include here? Please specify what needs to be included here.

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 9]
```diff
@@ -22,10 +22,10 @@ If the source file contains at least one entrypoint, it can be executed using th
 $ ballerina run foo.bal
 ```
 
-You can compile a source file with an entrypoint into a linked binary that has a `.jar` extension.
+You can compile a source file with an entrypoint into an executable jar.
```
Comment (Manuri) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
You can compile a source file with an entry point into an executable jar.
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md (hunk 2 above)
Comment (warunalakshitha) [reply website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]: Latest changes seems fine.

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 10]
```diff
@@ -483,9 +479,9 @@ $ kubectl get configmap
 NAME                                              DATA      AGE
 employee-data-service-ballerina-conf-config-map   1         2m
 ```
-This is the config-map created for the `ballerina.conf` file, as the `ballerinaConf:"./conf/data-service.toml"` attribute is used. At run time, it is an equivalent of:
+This is the config-map created for the `ballerina.conf` file, as the `ballerinaConf:"./conf/data-service.conf"` attribute is used. At run time, it is an equivalent of:
 ```bash
-$ ballerina run --config ./conf/data-service.toml <source>.balx 
+$ ballerina run --b7a.config.file=./conf/data-service.conf <source>.jar
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
$ ballerina run <source>.jar --b7a.config.file=./conf/data-service.conf
```

File: website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md [hunk 11]
```diff
@@ -483,9 +479,9 @@ $ kubectl get configmap
 NAME                                              DATA      AGE
 employee-data-service-ballerina-conf-config-map   1         2m
 ```
-This is the config-map created for the `ballerina.conf` file, as the `ballerinaConf:"./conf/data-service.toml"` attribute is used. At run time, it is an equivalent of:
+This is the config-map created for the `ballerina.conf` file, as the `ballerinaConf:"./conf/data-service.conf"` attribute is used. At run time, it is an equivalent of:
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn/how-to-deploy-and-run-ballerina-programs.md]:
```suggestion
This is the config-map created for the `ballerina.conf` file, as the `ballerinaConf:"./conf/data-service.conf"` attribute is used. At run time, it is equivalent to:
```

**Pull request merged**